{{ cookiecutter.package_name }}/docs/conf.py

```diff
@@ -52,7 +52,7 @@
 #needs_sphinx = '1.2'
 
 # To perform a Sphinx version check that needs to be more specific than
-# major.minor, call `check_sphinx_version("x.y.z")` here.
+# major.minor, call `check_sphinx_version("M.m.X")` here.
```

Comment:
Is this "M.m.X" convention documented anywhere? I am used to seeing people use "x.y.z" but I might have missed something.

Reply:
I've looked at the Python PEPs and M.m is a colloquialism. The formal usage is X.Y.Z, but always capitalized. Thanks for catching that. I've changed to caps